scripts/deploy-circle.sh

```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
 
-set -e
```

Comment:
Was this causing problems with the conditional somehow?

Reply:
I just moved it down below

Reply:
Yeah, I noticed. I was wondering why?

Reply:
Oh, I was just kinda following the example script from https://github.com/18F/18f-bot/blob/master/.circleci/deploy.sh where it had it inside of the conditional instead of at the top. I think it can be moved back though.

Reply:
Probably not an issue. I was just curious if there was something in the conditional that we were worried might break or cause some weird behavior to necessitate moving it.